hztb-common/src/main/java/com/svs/hztb/common/model/RequestData.java

```diff
@@ -5,9 +5,10 @@
 
 import com.google.common.collect.ImmutableMap;
 
+@FunctionalInterface
 public interface RequestData {
 	public String getRequestId();
-	
+
 	default Optional<Map<String, String>> getHeaders(String targetId, String mediaType) {
 		final String HTTP_TARGET_ID = "TARGET_ID";
 		final String HTTP_ACCEPT = "ACCEPT";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AS00117)